lapis2/src/test/kotlin/org/genspectrum/lapis/auth/ProtectedDataAuthorizationTest.kt

```diff
@@ -53,13 +56,15 @@ class ProtectedDataAuthorizationTest(@Autowired val mockMvc: MockMvc) {
     fun `given no access key in POST request to protected instance, then access is denied`() {
         mockMvc.perform(postRequestWithBody(""))
             .andExpect(MockMvcResultMatchers.status().isForbidden)
-            .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON))
             .andExpect(
-                MockMvcResultMatchers.content().json(
+                content().json(
                     """
```

Comment:
The error string appears ~6 times here and apparently is always the same, i.e., we dont test other errors here. We could extract the error string. Not only we would shorten the tests by a bit, the error that is tested here can be on the top of the file....